rasa_scheduler/celery_tasks.py

```diff
@@ -16,12 +24,21 @@
 def trigger_ask_foreseen_hrs(self):  # pylint: disable=unused-argument
     """Task to trigger RASA to set reminder for every user.
     """
-    # TODO_db: get user IDs from database
     # TODO: add Celery or http error handling
-    user_ids = ['38527']
+    user_ids = get_user_ids()
     for user in user_ids:
         endpoint = f'http://rasa_server:5005/conversations/{user}/trigger_intent'
         headers = {'Content-Type': 'application/json'}
         params = {'output_channel': 'niceday_input_channel'}
         data = '{"name": "EXTERNAL_trigger_ask_foreseen_hrs"}'
         requests.post(endpoint, headers=headers, params=params, data=data)
```

Comment:
If we have thousands of users, each with multiple reminders, is all of this going to be scalable? I guess < 10k is still small.

Reply:
No of course it is not scalable. Luckily we have only 500 users max (and likely not all active at the same time). See also [the non-functional requirements in the design](https://github.com/PerfectFit-project/virtual-coach-server/blob/main/docs/design.md).